libr/util/log.c

```diff
@@ -31,3 +29,24 @@ R_API void r_log_error(const char *str) {
 R_API void r_log_progress(const char *str, int percent) {
 	printf ("%d%%: %s\n", percent, str);
 }
+
+R_API void r_vlog(RLogLevel level, const char *fmt, va_list ap) {
+	static const char *headers[R_LOG_MAX_VALUE] = {
+		[R_LOG_DEBUG] = "DEBUG: ",
+		[R_LOG_INFO] = "INFO: ",
+		[R_LOG_WARNING] = "WARNING: ",
+		[R_LOG_ERROR] = "ERROR: ",
+		[R_LOG_CRITICAL] = "CRITICAL: ",
+	};
+
+	const char *hdr = level < R_LOG_MAX_VALUE ? headers[level] : "";
+	eprintf (hdr);
```

Comment:
wrong use of eprintf, first argument must be a format string, aka "%s"

Reply:
it is a format string. it's a `const char *`, not a `char *`.